### ISSUE TITLE
Update bundler to version 2.0

### ISSUE THREAD

**Issue by rhuanbarreto** [docker-library/ruby#268]
Need to update bundler on all images to version 2.0

**Comment by tianon** [comment]
See #246 and #255.

**Comment by rhuanbarreto** [comment]
But if this was removed, how should I proceed?

**Comment by tianon** [comment]
If you need a newer Bundler version, I'd recommend running something like `gem install bundler --version X.Y.Z --force`, although all the images should currently contain 2.0.x.

**Comment by yosifkit** [comment]
@tianon was off; all of the images are fixed at rubygems 3.0.1 (or newer if it comes with Ruby source), which includes bundler `1.17.2`. We do not have any images with Bundler `2.0.x`. If you need a newer version of Bundler or gems, you should install it.

```console
$ bashbrew list --uniq ruby | xargs -n1 -I{} docker run --rm {} gem --version
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
3.0.1
$ bashbrew list --uniq ruby | xargs -n1 -I{} docker run --rm {} bundle --version
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
Bundler version 1.17.2
```
